### PR TITLE
Add Language colors for the repositories.

### LINKED ISSUE #24: Add repository language badge to repo-card
It would be neat to have the repository language information displayed on the repository cards, just like it is in Github.

screenshot 2019-02-22 at 17 09 45

The repository API publishes that information in the form of a string, so to replicate GitHub's behavior I've had to write some ugly code by adding an if statement for each language:

```html
    {% if repository.language == "Go" %}
    <span class="repo-language-color mr-1" style="background-color:#00ADD8;"></span>
    <span class="ml-0" itemprop="programmingLanguage">Go</span>
    {% endif %}
```

It would be neat if somehow it was possible to have an equivalent to this that doesn't add much complexity, but I'm not sure if it's doable.

### PR DESCRIPTION
**Pull request by ccsplit**
Used colors.json from https://github.com/ozh/github-colors/blob/master/colors.json
to get the latest colors for each of the languages and then added to the repo-card
with the language and a circle for the color.

Resolves #24.

### CODE REVIEW

File: _includes/repo-card.html [hunk 1]
```diff
@@ -20,5 +20,9 @@ <h1 class="f4 lh-condensed mb-1">
         <svg class="octicon octicon-git-branch mr-1" viewBox="0 0 10 16" version="1.1" width="10" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M10 5c0-1.11-.89-2-2-2a1.993 1.993 0 0 0-1 3.72v.3c-.02.52-.23.98-.63 1.38-.4.4-.86.61-1.38.63-.83.02-1.48.16-2 .45V4.72a1.993 1.993 0 0 0-1-3.72C.88 1 0 1.89 0 3a2 2 0 0 0 1 1.72v6.56c-.59.35-1 .99-1 1.72 0 1.11.89 2 2 2 1.11 0 2-.89 2-2 0-.53-.2-1-.53-1.36.09-.06.48-.41.59-.47.25-.11.56-.17.94-.17 1.05-.05 1.95-.45 2.75-1.25S8.95 7.77 9 6.73h-.02C9.59 6.37 10 5.73 10 5zM2 1.8c.66 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2C1.35 4.2.8 3.65.8 3c0-.65.55-1.2 1.2-1.2zm0 12.41c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zm6-8c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2z"></path></svg>{{ repository.forks_count }}
       </a>
     {% endif %}
+    {% if repository.language %}
+      <span class="repo-language-color mr-1" style="background-color:{{ site.data.colors[repository.language]["color"] }}"></span>
+      <span class="ml-0" itemprop="programmingLanguage">{{ repository.language }}</span>
```
Comment (kunicmarko20) [inline _includes/repo-card.html]: would just add `mr-3` to class here, to match the stars and forks

Comment (ccsplit) [reply _includes/repo-card.html]: @kunicmarko20, changed the order to match GitHub and also added `mr-3` to the language to match the stars and forks.

Comment (kunicmarko20) [reply _includes/repo-card.html]: Looks good now :ok_hand:

**Comment by Ullaakut** [comment]
Actually, not sure why but with the same CSS style as the one used on GitHub, the language colors and labels are not aligned by one pixel 🤔 Changing their `top` attribute to 2px instead of 1 seems to fix the issue, but I'm not sure where this comes from.

**Comment by ccsplit** [comment]
Yeah it does look like they are a little off at least with the force bottom changes I adapted from yours on my GitHub page. I'll have to check and make sure this is the same case with the default repo, if I get a chance tomorrow. Not sure if you were applying the changes to your current GitHub page or the default layout.

File: assets/styles.scss [hunk 2]
```diff
@@ -17,6 +17,6 @@ body {
     display: inline-block;
     height: 12px;
     position: relative;
-    top: 1px;
+    top: 2px;
```
Comment (Ullaakut) [inline assets/styles.scss]: 👍🏽

**Comment by GodlikePenguin** [comment]
This change was regressed in https://github.com/github/personal-website/commit/15cf219abea96fef0e486bef754f15f60732bfa3#diff-a2e39333fa07588cea121325f3212474

**Comment by brandonrosage** [comment]
My bad, @GodlikePenguin. I restored the feature in a commit this morning 👍 Thanks for catching this.

**Pull request merged**